doc/api/net.md

```diff
@@ -78,19 +137,29 @@ server.listen(() => {
 });
 ```
 
-Don't call `server.address()` until the `'listening'` event has been emitted.
-
 ### server.close([callback])
 <!-- YAML
 added: v0.1.90
 -->
 
-Stops the server from accepting new connections and keeps existing
-connections. This function is asynchronous, the server is finally
-closed when all connections are ended and the server emits a [`'close'`][] event.
-The optional `callback` will be called once the `'close'` event occurs. Unlike
-that event, it will be called with an Error as its only argument if the server
-was not open when it was closed.
+* `callback` {Function}
+
+The `server.close()` method stops the server from accepting new connections
+while maintaining existing connections that have not yet completed. Once the
+existing connections have closed, the server will emit the [`'close'`][]
```

Comment:
you  changed "ended" to "closed", but neither are quite correct: socket reset will also cause this to happen. Not sure what verb to use, though, perhaps "no longer open"?